tests/test_dispatcher.py

```diff
@@ -174,6 +175,40 @@ def test_error_handler_that_raises_errors(self, dp):
 
         assert self.count == 1
 
+    def test_default_run_async_error_handler(self, dp, monkeypatch):
```

Comment:
okay, the tests now test the proper thing :) let's improve them a bit more:

* please also test that passing `Defaults(run_async=False)` does not run stuff asynchorously (you can use `pytest.mark.parametrize` here, you can have a look at other tests that use that
* please do something like 

```python
dp.bot.defaults = Defaults(run_async=True)
try:
    # the actual test
finally:
    dp.bot.defaults = None
```

  to ensure that the defaults are removed at the end of the test, even if it fails